pippo-session-parent/pippo-session-mongodb/src/test/java/ro/pippo/session/mongodb/MongoDBSessionDataStorageTest.java

```diff
@@ -50,10 +49,10 @@ public static void setUpClass() throws IOException {
         mongodExe = starter.prepare(
                 new MongodConfigBuilder()
                 .version(Version.Main.PRODUCTION)
-                .net(new Net(PORT, Network.localhostIsIPv6()))
+                .net(new Net(Network.getFreeServerPort(), Network.localhostIsIPv6()))
                 .build());
         mongod = mongodExe.start();
-        mongoClient = new MongoClient(HOST, PORT);
+        mongoClient = new MongoClient(HOST, Network.getFreeServerPort());
```

Comment:
I don't know if the second call of `Network.getFreeServerPort()` returns the same port (another variant is to use an internal variable `port = Network.getFreeServerPort()`). 
If you are sure that the actual code is OK I will merge it.